app/javascript/packs/users-table.jsx

```diff
@@ -1,34 +1,96 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import axios from "axios";
-import Table from "../components/table";
+import axios from 'axios';
+
 
 class UsersTable extends React.Component {
   compare = (a, b) => (b > a) - (b < a);
 
-  state = {
-    users: []
-  };
+    
+    
+    state = {
+        users: [],
+        following: []
+    };
+
+    componentWillMount() {
+        axios.get('/api/v1/users.json', {}, { 'Content-Type': 'application/json' })
```

Comment:
Use `/api/v1/users` instead of `/api/v1/users.json` because this controller accepts json by default. :wink: 
You could also look into concurrent requests with [axios](https://github.com/axios/axios) but don't feel obliged to.